src/ILVerify/src/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("ILVerify.Tests")]
```

Comment:
Agree, that would be nice and I’m sure we want to have a .dll flavor of ILVerify. I wanted to test without `InternalsVisibleTo`, but currently there is no API surface, everything is internal. So I think, the first step is to define what we want to be public. I think we need `VerifierError`, `VerificationErrorArgs`, and most importantly some methods which are currently on `ILImporter` (or we create a façade class for that?).

Do we want to do that here? Or create an issue, define it there and I do it in another PR.